integreat_compass/cms/models/offers/language.py

```diff
@@ -28,7 +28,7 @@ def __str__(self):
         :return: A readable string representation of the page
         :rtype: str
         """
-        return f"Language: {self.english_name}"
+        return self.english_name
```

Comment:
```suggestion
        return _(self.english_name)
```
Why not translate it?